gravatar-quickeditor/src/main/java/com/gravatar/quickeditor/ui/components/SelectableAvatar.kt

```diff
@@ -0,0 +1,57 @@
+package com.gravatar.quickeditor.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.gravatar.quickeditor.R
+import com.gravatar.ui.GravatarTheme
+
+@Composable
+private fun SelectableAvatar(
+    imageUrl: String,
+    isSelected: Boolean,
+    onAvatarClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    val cornerRadius = 7.dp
```

Comment:
I think this one will be increased based on the current design iteration but let's wait for the final one.